serverless-scheduler-proxy/cloudbuild.yaml

```diff
@@ -35,6 +35,8 @@ steps:
     id: "serverless-scheduler-proxy-deploy"
     waitFor: ["serverless-scheduler-proxy-push"]
     entrypoint: bash
+    env:
+      - "SERVICE_IDENTIFIER=c7mcaca6sa"
```

Comment:
It seems like the same identifier is shared between Cloud Run deployments. Is it guaranteed?

Reply:
In some docs it's referred to as the "project hash". From all the examples I see online, it's static for your project, but I don't see any official declaration on the product docs.

Reply:
Good for now. Maybe give feedback to the product team?